plugins/callback/loganalytics.py

```diff
@@ -0,0 +1,252 @@
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    callback: loganalytics
+    type: aggregate
+    short_description: Posts task results to Azure Log Analytics
+    author: "Cyrus Li <cyrus1006@gmail.com>"
+    description:
+      - This callback plugin will post task results in JSON formatted to an Azure Log Analytics workspace.
+      - Credit to authors of splunk callback plugin(splunk.py).
+    version_added: "2.9"
+    requirements:
+      - Whitelisting this callback plugin.
+      - An Azure log analytics work space has been established.
+      - Define the Azure log analytics workspace id and key in ansible.cfg
+    options:
+      workspace_id:
+        description: Workspace ID of the Azure log analytics workspace
+        env:
+          - name: WORKSPACE_ID
+        ini:
+          - section: callback_loganalytics
+            key: workspace_id
+      shared_key:
+        description: Shared key to connect to Azure log analytics workspace
+        env:
+          - name: WORKSPACE_SHARED_KEY
+        ini:
+          - section: callback_loganalytics
+            key: shared_key
+'''
+
+EXAMPLES = '''
+examples: >
+  Whitelist the plugin
+    [defaults]
+    callback_whitelist = loganalytics
```

Comment:
```suggestion
    callback_whitelist = community.general.loganalytics
```
Otherwise it won't work.